app/main/posts/collections/collections-controller.js

```diff
@@ -39,8 +39,17 @@ module.exports = [
 
         // Extend filters, always adding the current collection id
         var extendFilters = function (filters) {
-            filters = angular.copy(filters, { set : [] });
+            //filters = angular.copy(filters, { set : []});
+            filters.set = [];
             filters.set.push(collection.id);
+            //Ensure that ALL posts are visible under collections
+            // Set default collection status filters
+            if (_.has(filters, 'status')) {
```

Comment:
This is problematic because it will overwrite the status filter on every update, not just the first load.

Need to split this into 1. function to always add collection to filters 2. setting the initial filters to include archived.. maybe just use `PostFilters.setFilters({ status: ['archive', 'draft', 'published'] });` rather than `PostFilters.clearFilters()`